NimbleMedium/Sources/Presentation/Modules/SideMenu/SideMenuActionsView.swift

```diff
@@ -18,8 +22,13 @@ struct SideMenuActionsView: View {
                     text: Localizable.menuOptionLogin(),
                     iconName: R.image.iconLogin.name
                 ) {
-                    print("Login button was tapped")
+                    onSelectOptionItem?()
+                    isShowingLoginScreen.toggle()
+                }
+                .fullScreenCover(isPresented: $isShowingLoginScreen) {
+                    LoginView(viewModel: factory.loginViewModel())
```

Comment:
We should have a view model for SideMenuActionsView to return an output with loginViewModel instead. And the SideMenuActionsViewModel will init an factory, in SideMenuActionsView, we don't need a factory.
```suggestion
                    LoginView(viewModel: viewModel.output.loginViewModel)
```

Reply:
oh I get your point, let me try and experiment it